index.html

```diff
@@ -1,220 +1,218 @@
-          <a href="" class="logo">chester</a>
-          <a href="" class="union-icon"><img src="./images/Union.png" alt=""></a>
-        </nav>
-      </header>
-      <section class="headline-section">
-        <div class="text-area">
-          <h1>I'm rotshidzwa <br>Glad to see <br>you!</h1>
-          <p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience!
-            If you like what you see and have a project you need coded, don’t hestiate to contact me.
-          </p>
-        </div>
-        <ul class="media-icon ">
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/twitter.png " alt="twitter ">
-                </a>
-            </li>
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/Linkedin-icon.png " alt="linkedin ">
-                </a>
-            </li>
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/medium.png " alt="medium ">
-                </a>
-            </li>
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/git.png " alt="git ">
-                </a>
-            </li>
-            <li>
-                <a href="# " class="icons ">
-                    <img src="./images/engelist.png " alt="angelist ">
-                </a>
-            </li>
-        </ul>
+<html  lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="style.css">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://kit.fontawesome.com/1dea6ac3ad.js" crossorigin="anonymous"></script>
+        <title>Portfolio</title>
+    </head>
+    <body>
+        <section class="Section-1">
+        <header id="Header"> 
+            <h3>  <a href="#" id="logo" class="desi">Chester</a></h3> 
+               <nav>
+                <ul class="desktop desi">
+                    <li class="nev-list">About</li>
+                    <li class="nev-list">Portfolio</li>
+                    <li class="nev-list">Contact</li>
+                </ul>
+                <i class="fa-solid fa-bars" id="Menu_id"></i>
+               </nav>
+        </header>
+        <section class="header-section">
+            <h1>I’m Rotshi,
+                <br> Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need 
+                coded, don’t hestiate to contact me.</p>
+            <h3>LET’S CONNECT</h3>
```

Comment:
- [ ] Please make sure this section displays properly to match the Figma design. This is caused by setting large fixed padding right which is too large on width `768px`.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/161294117-259f2efa-473c-44ab-83bc-83fb72409526.png) | ![image](https://user-images.githubusercontent.com/62290545/161294253-1e01e02b-2a37-4b6b-9d36-e68113ab8e2f.png) |